DataStructures/Graph.java

```diff
@@ -0,0 +1,105 @@
+import java.io.*;
+import java.util.*;
```

Comment:
Please avoid using '*' in your imports. Rather import the each package explicitly.

Reply:
Okay sir I will correct it.

Reply:
Mind every suggested change at other instances of your codes as well.
This will surely help you to write better codes in future.

Reply:
Okay sir sure I will